backend/services/.env.dev

```diff
@@ -15,7 +15,6 @@ DISABLE_LOW_PRIORITY_EMAIL=false
 ASYNC_QUEUE_NAME=https://sqs.us-east-1.amazonaws.com/302213478610/AsyncQueuedev.fifo
 DOMAIN_MAP=false
 EXPIRES_IN=3600
-SYNC_ENABLE=true
```

Comment:
Don't we need to enable SYNC in test system?

Reply:
it is duplicated